src/share/vm/gc_interface/allocTracer.inline.hpp

```diff
@@ -117,4 +118,5 @@ inline void AllocTracer::send_opto_fast_allocation_event(KlassHandle klass, oop
   thread->trace_data()->incr_alloc_count_until_sample(interval);
 }
 
+#endif
```

Comment:
this `#endif` is too far away from `#if`, pls add comments to keep good readability
`+#endif // #if INCLUDE_TRACE`

Reply:
fixed